src/views/tables/PositionsTable/PositionsTriggersCell.tsx

```diff
@@ -57,7 +57,7 @@ export const PositionsTriggersCell = ({
         type={ButtonType.Link}
         action={ButtonAction.Navigation}
         size={ButtonSize.XSmall}
-        onClick={onViewOrdersClick}
+        onClick={() => onViewOrdersClick(market)}
       >
         {stringGetter({ key: STRING_KEYS.VIEW_ORDERS })}
         {<Icon iconName={IconName.Arrow} />}
```

Comment:
visual nit: can we reduce font weight / make it smaller hehe

Reply:
ahh do you mean on the arrow next to viewOrders? yeah it looks a lil heavy n clunky but I think it's because the SVG is drawn that thickly :'( (and without a strokeWidth). actually, lemme upload a new arrowIcon svg with a customizable `strokeWidth`